invokeai/frontend/web/src/features/stylePresets/components/StylePresetPromptField.tsx

```diff
@@ -0,0 +1,63 @@
+import { Flex, FormControl, FormLabel, IconButton, Textarea } from '@invoke-ai/ui-library';
+import { PRESET_PLACEHOLDER } from 'features/stylePresets/hooks/usePresetModifiedPrompts';
+import type { ChangeEventHandler } from 'react';
+import { useCallback, useMemo, useRef } from 'react';
+import type { UseControllerProps } from 'react-hook-form';
+import { useController } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { PiBracketsCurlyBold } from 'react-icons/pi';
+
+import type { StylePresetFormData } from './StylePresetForm';
+
+interface Props extends UseControllerProps<StylePresetFormData> {
```

Comment:
```suggestion
interface Props extends UseControllerProps<StylePresetFormData, 'positivePrompt' | 'negativePrompt'> {
```

And remove the casts